skietbaan_fe/src/components/Login.js

```diff
@@ -28,32 +27,29 @@ class App extends Component {
      }
     if (!validatePassword(this.state.passwordValue)) {
       stateUpdate.invalidPassword= true;
+      isValid = false;
     };
-    isValid = false;
     if (!validateEmail(this.state.emailValue)) {
       stateUpdate.invalidEmail= true;
+      isValid = false;
     }; 
-    isValid = false; 
     if (validateUsername(this.state.usernameValue)) {
       stateUpdate.invalidUsername= true;
+      isValid = false;
     };
-    isValid = false;
-    
     this.setState({
       ...stateUpdate ,
       validForm: isValid
     });
-  }
-
+  };
   Login() {
     if (this.state.validForm) {
       let RequestObject = {
         "Username": this.state.usernameValue,
-        "Password": this.state.emailValue,
-        "Surname": this.state.passwordValue
+        "Email": this.state.emailValue,
+        "Password": this.state.passwordValue
```

Comment:
This should ideally be sent up hashed already. This is to ensure that if the connection is insecure or has been compromised the password is at least not sent as plain text.

Reply:
Alright fixed